src/rpc/misc.cpp

```diff
@@ -366,6 +367,27 @@ static UniValue setmocktime(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+static UniValue mockscheduler(const JSONRPCRequest& request)
+{
+        RPCHelpMan{"mockscheduler",
+            "\nBump the scheduler into the future (-regtest only)\n",
+            {
+                {"delta_time", RPCArg::Type::NUM, RPCArg::Optional::NO, "Number of seconds to forward the scheduler into the future." },
+            },
+            RPCResults{},
+            RPCExamples{""},
+        }.Check(request);
+
+    if (!Params().MineBlocksOnDemand())
```

Comment:
is there a better way to query if we're regtest?

Reply:
I was wondering that too. This is I found but it surprised me.